oxfordpython/settings.py

```diff
@@ -17,7 +17,13 @@
 
 
 # Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'y54h%=s#!+=#-oahz$spl9lhu(*c0y7@+^*c0)q+9g=7)5-9=8'
```

Comment:
We should configure this with an environment variable. https://www.mattlayman.com/2017/12-factor-django-apps.html